dipy/align/imwarp.py

```diff
@@ -96,310 +97,6 @@ def get_direction_and_spacings(affine, dim):
     A = affine[:dim,:dim]
     return A.dot(np.diag(1.0/scalings)), scalings
 
-
-class ScaleSpace(object):
```

Comment:
I moved the ScaleSpace class to its own module because now we have two different ways of building a scale space (the ANTS-like anisotropic scale space and the ANTS2-like isotropic one). The new scale space is IsotropicScaleSpace.